Classes/ResourceServer/GitLab.php

```diff
@@ -170,16 +169,13 @@ public function loadUserDetails(ResourceOwnerInterface $user): void
             $accessLevel = max($accessLevel, $project['permissions']['project_access']['access_level'] ?? 0);
             $accessLevel = max($accessLevel, $project['permissions']['group_access']['access_level'] ?? 0);
 
-            $sharedGroups = $this->sharedGroupsForProject($gitlabClient, $project);
-            foreach ($sharedGroups as $sharedGroupId) {
-                try {
-                    $member = $gitlabClient->groups()->member($sharedGroupId, $user->getId());
-                    if ($member) {
-                        $accessLevel = max($accessLevel, $member['access_level'] ?? 0);
-                    }
-                } catch (\Exception $exception) {
-                    // user has no access to see details
+            try {
```

Comment:
This try-catch-block is also superfluous as it adds no value above the outer try-catch-block